scss/_variables.scss

```diff
@@ -1150,7 +1150,9 @@ $dropdown-item-padding-y:           $spacer * .25 !default;
 $dropdown-item-padding-x:           $spacer !default;
 
 $dropdown-header-color:             $gray-600 !default;
-$dropdown-header-padding:           $dropdown-padding-y $dropdown-item-padding-x !default;
+$dropdown-header-padding-x:         $dropdown-item-padding-x !default;
+$dropdown-header-padding-y:         $dropdown-padding-y !default;
+$dropdown-header-padding:           $dropdown-header-padding-y $dropdown-header-padding-x !default;
```

Comment:
This var becomes unused so can be dropped.
```suggestion
```

Reply:
I think we should deprecate it but we can't drop it IMHO, could cause compilation issue for folks using it in their custom build.

Reply:
Yes my bad. You're totally right.